src/Badge.tsx

```diff
@@ -66,3 +66,5 @@ const badgeStyles = ({
     },
   ];
 };
+
```

Comment:
looks like you missed a displayName?

Reply:
It's not required in this specific case because it's a simple `FunctionComponent` and not a `forwardRef`